src/Helper/Customer/Utility.php

```diff
@@ -24,21 +24,26 @@
 
 use Magento\Customer\Api\Data\CustomerInterface;
 use Magento\Customer\Model\Data\Address;
+use Magento\Framework\Api\CustomAttributesDataInterface;
```

Comment:
Move it down right before that line to group by directory
> use Magento\Framework\Api\SimpleDataObjectConverter;

Reply:
done